Projects/Server/Items/Item.cs

```diff
@@ -4123,6 +4124,21 @@ public virtual void OnAosSingleClick(Mobile from)
         }
     }
 
+    public static void AppendWithSpace(StringBuilder builder, string text)
```

Comment:
This should be moved to one of the text handling files. I'll need to look it up later to give more details.